SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -352,5 +372,70 @@ private Task SendErrorAttachmentsAsync(Guid errorId, IEnumerable<ErrorAttachment
             }
             return Task.WhenAll(tasks);
         }
+
+        private ErrorReport BuildErrorReport(ManagedErrorLog log)
+        {
+            if (_errorReportCache.ContainsKey(log.Id))
+            {
+                return _errorReportCache[log.Id];
+            }
+            var file = ErrorLogHelper.GetStoredExceptionFile(log.Id);
+            if (file == null)
+            {
+                return null;
+            }
+            var exception = ErrorLogHelper.ReadExceptionFile(file);
```

Comment:
What happens if `exception` is null?

Reply:
Then the Exception property will be null on the ErrorReport.
This also happens on Xamarin when we can't deserialize.